go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -101,26 +112,34 @@ func NewEmergencyReparenter(ts *topo.Server, tmc tmclient.TabletManagerClient, l
 // keyspace and shard.
 func (erp *EmergencyReparenter) ReparentShard(ctx context.Context, keyspace string, shard string, opts EmergencyReparentOptions) (*events.Reparent, error) {
 	var err error
+	statsLabels := []string{keyspace, shard}
 	opts.lockAction = erp.getLockAction(opts.NewPrimaryAlias)
 	// First step is to lock the shard for the given operation, if not already locked
 	if err = topo.CheckShardLocked(ctx, keyspace, shard); err != nil {
 		var unlock func(*error)
 		ctx, unlock, err = erp.ts.LockShard(ctx, keyspace, shard, opts.lockAction)
 		if err != nil {
+			legacyERSFailureCounter.Add(1)
+			ersFailureCounter.Add(statsLabels, 1)
 			return nil, err
 		}
```

Comment:
This is still incorrect...
We are incrementing the failure counter and returning from the function without increasing the total count. So the total count would still not be the sum of successes and failures.
Also we shouldn't be changing the behaviour of the legacy counters.

Reply:
@GuptaManan100 makes sense, updated 👍

As for the `DCO` CI failure, it seems the rebase suggested by the failed check is getting rejected for reasons I cannot explain:
```
tvaillancourt@tvailla-ltmxctu vitess % git push --force-with-lease origin stats-reparent_shard_operation_timings
To https://github.com/timvaillancourt/vitess
 ! [rejected]              stats-reparent_shard_operation_timings -> stats-reparent_shard_operation_timings (stale info)
error: failed to push some refs to 'https://github.com/timvaillancourt/vitess'
```
I'm curious if you had ideas here 🤔

Reply:
I have no idea! Maybe @deepthi knows?

Reply:
@GuptaManan100 I managed to resolve it by squashing my commits + signing 🎉. Thanks for checking!